views/js/runner/testStore.js

```diff
@@ -190,30 +200,32 @@ define([
                             },
 
                             /**
-                            * Set an item with the given key
-                            * @param {String} key - the item key
-                            * @param {*} value - the item value
-                            * @returns {Promise<Boolean>} with true in resolve if added/updated
-                            */
+                             * Set an item with the given key
+                             * @param {String} key - the item key
+                             * @param {*} value - the item value
+                             * @returns {Promise<Boolean>} with true in resolve if added/updated
+                             */
                             setItem : function setItem(key, value){
+                                trackChange();
```

Comment:
if I have `this.getItem('i') === 1` and I do `this.setItem('i', 1)` then I have storage changed state.

Reply:
Yes, the goal of the change tracking is to know if a store have been changed. 
Since there's a lot of `setItem` called I don't want to always perform a `getItem` before and compare the value and think it's better to consider the changes  when a write operation occur, no matter what it contain.